src/ExtensionsCommon/DataItemContainer.h

```diff
@@ -47,12 +47,12 @@ class ATL_NO_VTABLE CDataItemContainer :
 		return  this->m_dataItem;
 	}
 
-	HRESULT FinalConstruct()
+	static HRESULT FinalConstruct()
```

Comment:
>static HRESULT FinalConstruct() [](start = 1, length = 31)

same comment here. Make sure that this isn't a public interface. It doesn't look like it is, otherwise it should override, and be stdcall, but that might be a mistake, too.